docs/src/views/Layout.tsx

```diff
@@ -0,0 +1,212 @@
+import * as React from 'react'
+import { Button, Flex, Header, Icon, Segment, Divider } from '@stardust-ui/react'
+
+import CodeSnippet from '../components/CodeSnippet'
+import ExampleSnippet from '../components/ExampleSnippet'
+import DocPage from '../components/DocPage'
+import GuidesNavigationFooter from '../components/GuidesNavigationFooter'
+
+const code = value => <code>{value}</code>
+
+const link = (content, href, isExternal = false) => (
+  <a href={href} {...isExternal && { target: 'blank' }}>
+    {content} {isExternal ? <Icon name="external" size="small" link fitted /> : ''}
+  </a>
+)
+
+const links = {
+  flex: link('Flex', '/components/flex'),
+  flexItem: link('Flex.Item', '/components/flex'),
+  grid: link('Grid', '/components/grid'),
+  segment: link('Segment', '/components/segment'),
+}
+
+export default () => (
+  <DocPage title="Layout">
+    <Header as="h2">Overview</Header>
+    <p>The following components are introduced by Stardust to handle layout aspects:</p>
+    <ul>
+      <li>{links.flex} - designed for layout in one dimension, either a row or a column.</li>
+      <li>
+        {links.grid} - designed for two-dimensional layout, rows and columns at the same time.
+      </li>
+    </ul>
+    <Header as="h3">Flex vs Grid</Header>
+    <p>
+      Sometimes it might not be evident if some particular case is a case of 1D (i.e. {code('Flex')}
+      ) or 2D (i.e. {code('Grid')}) layout. As a result, quite often {code('Grid')} component is
+      used at places where it is sufficient to use {code('Flex')}.
+    </p>
+    <p>Here is the question you might ask yourself to decide on that:</p>
+    <ul>
+      <li>
+        do I only need to control the layout by row <strong>or</strong> column? – if yes, use{' '}
+        {code('Flex')}.
+      </li>
+      <li>
+        do I need to control the layout by row <strong>and</strong> column? – if yes, use{' '}
+        {code('Grid')}.
+      </li>
+    </ul>
+    <Header as="h3">Note on Segment component</Header>
+    <p>
+      {links.segment} component may be abused for layout purposes, like in the following example:
+    </p>
+    <CodeSnippet
+      label="AbsolutePositioned.jsx"
+      value={`
+        import { Segment } from '@stardust-ui/react'
+
+        const OpaqueBackground = ({ children, color }) =>
+            <Segment styles={{ backgroundColor: color }} content={{ children }} />
+
+        const RelativePositioned = ({ children, top, right, bottom, left }) => (
+            <Segment styles={{ position: 'relative', top, right, bottom, left }} content={children} />
+        )
+      `}
+    />
+    <ExampleSnippet
+      render={() => {
+        const OpaqueBackground = ({ children, color }) => (
+          <Segment styles={{ backgroundColor: color }} content={{ children }} />
+        )
+
+        return (
+          <OpaqueBackground color="red">
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/54830163-a2300a80-4cc0-11e9-9718-8f270852c267.png)


nit: Can use we less toxic color? 🌞 It triggers all my focus to it

Reply:
sure :)

Reply:
![image](https://user-images.githubusercontent.com/9024564/54907063-f74c6600-4ee4-11e9-93f2-0679103fa9ac.png)